feature/google/src/main/java/com/revenuecat/purchases/google/PurchaseContext.kt

```diff
@@ -0,0 +1,11 @@
+package com.revenuecat.purchases.google
+
+import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.GoogleProrationMode
+
+class PurchaseContext(
+    val productType: ProductType,
+    val presentedOffering: String?,
+    val subscriptionOptionSelected: String?,
```

Comment:
Should we rename these two to `presentedOfferingId` and `selectedSubscriptionOptionId`?

Reply:
Sure